www/cocoa-rest-client/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+PortGroup                   github 1.0
+
+github.setup                mmattozzi cocoa-rest-client 1.4.5
+github.tarball_from         releases
+revision                    1
+
+categories                  www
+platforms                   darwin
+supported_archs             x86_64
+license                     BSD
+maintainers                 {@lhaeger} openmaintainer
+
+description                 A free, native macOS app for testing HTTP/REST endpoints
+long_description            Test and interact with HTTP/REST resources:\
+                            - Make GET, PUT, POST, DELETE, and HEAD calls.\
+                            - Set request headers and display response headers.\
+                            - Automatically pretty print XML and JSON content.\
+                            - Quick save of request URLs, body, and headers in one convenient drawer.\
+                            - Lightweight: Low real memory usage.\
+                            - SSL Support (including untrusted certificates).\
+                            - Optionally follows HTTP redirects.\
+                            - Set HTTP Basic or Digest Auth.\
+                            - Set HTTP request body content to a raw text blob or a list of parameters.\
+                            - Upload files using HTTP multipart requests. HTTP form encoding also supported.\
+                            - Unified diff tool for comparing responses.
+
+homepage                    http://mmattozzi.github.io/cocoa-rest-client/
+
+distname                    CocoaRestClient-${version}
+extract.suffix              .dmg
+
+checksums                   rmd160  0136fcc44f38bf9df4e0270e6f6e2de6b4cdac06 \
+                            sha256  dccc638d7b8e9f793055cb1c4fc49da9f0f35e24e89f3559e53810150c77a6b1 \
+                            size    6070132
+
+pre-fetch                   {
+                            if { ${os.major} <= 15 } {
+                                    return -code error "This portrequires at least macOS Sierra (10.12)."
+                                    }
+                            }
+
+extract                     {
+                            file mkdir /tmp/${distname}
+                            system "hdiutil attach ${distpath}/${distfiles} -private -nobrowse -mountpoint /tmp/${distname}"
+                            }
+
+use_configure               no
+
+build                       {}
+
+destroot                    {
+                            copy /tmp/${distname}/CocoaRestClient.app ${destroot}${applications_dir}
```

Comment:
probably more efficient to move the app here, than make a copy ?

Reply:
I think I tried that first and ran into something, just do not remember right now. Will check again...

Reply:
Hmm, would be good to know what that is. Permissions maybe ?? 
copy is OK if move fails, just means potentially more I/O than really needed...

Reply:
Just realised why move likely cannot work here. Its because this is a mounted dmg.....
So forget this. Copy is the right option...